filetracker/__init__.py

```diff
@@ -359,12 +360,18 @@ def delete_file(self, name):
 
     def list_files(self):
         result = []
-        for root, dirs, files in os.walk(self.dir):
+        for root, _dirs, files in os.walk(self.dir):
             for basename in files:
                 relative_dir = os.path.relpath(root, self.dir)
-                relative_basename = os.path.join(relative_dir, basename)
-                result.append((relative_basename, os.path.getmtime(
-                    os.path.join(root, basename))))
+                store_full_dir = ('/' if relative_dir == '.'
+                                  else '/' + relative_dir)
+                store_full_basename = os.path.join(store_full_dir, basename)
```

Comment:
full_path to jest pełna ścieżka w systemie, potrzebna dla ```os.stat```. store_full_basename to poprawna nazwa pliku, na której operuje filetracker (ona musi zaczynać się od backslasha). Można to zamienić na ```store_full_dir = normapth(join('/', relative_dir))```